frontend/packages/shipwright-plugin/src/components/build-list/BuildTable.tsx

```diff
@@ -93,28 +96,28 @@ export const BuildRow: React.FC<RowFunctionArgs<Build, CustomData>> = ({
         <ResourceLink kind="Namespace" name={build.metadata.namespace} />
       </TableData>
       <TableData className={columnClassNames[2]}>
-        {latestBuildRun ? (
+        {build.latestBuild ? (
           <ResourceLink
             kind={buildRunKindReference}
-            name={latestBuildRun.metadata?.name}
-            namespace={latestBuildRun.metadata?.namespace}
+            name={build?.latestBuild?.metadata?.name}
+            namespace={build?.latestBuild?.metadata?.namespace}
```

Comment:
Nit: build is always defined, and you check that build.latestBuild exists also above:

```suggestion
            name={build.latestBuild.metadata?.name}
            namespace={build.latestBuild.metadata?.namespace}
```